### PR TITLE
Seb2: JSDoc Coverage

### PR DESCRIPTION
**Pull request by SebastianSafaiRad**
Started working on doc strings using JSDoc. Got a good start on all of the files under the views category.  In order to use JSDoc you must install globally using "npm install -g jsdoc".  Then you can run using "jsdoc file.js".  Doc Strings using This merge should have no conflicts

### CODE REVIEW

**Review by JonathanGrant92 — CHANGES_REQUESTED** [review]
Please add 'views/out/` to the top of the .gitignore file. Then run 'git rm --cached -r views/out/`. Then commit and push to repo/Seb2 again.

**Comment by JonathanGrant92** [comment]
Please don't run `npm i -g jsdoc` because jsdoc is a development dependency. Run`npm i` to update project environment with newest `package.json`. Developer tools should be installed either as `npm i -g --save-dev jsdoc` or `npm i --save-dev jsdoc`.  The `-g` is the global flag, if you want to install globally, not in a specific project environment.

**Review by JonathanGrant92 — APPROVED** [review]
Looks good!

File: views/Home.js [hunk 1]
```diff
@@ -23,6 +24,13 @@ const Home = () => {
     }
   }
```
Comment (JonathanGrant92) [inline views/Home.js:26]: Missing `}`. Sorry, I accidentally did this when merging conflicts!

**Review by NoelC9 — APPROVED** [review]
Thanks for adding comments.

**Pull request merged**